main.go

```diff
@@ -19,6 +19,7 @@ import (
 	"bufio"
 	"flag"
 	"fmt"
+	"html/template"
```

Comment:
Did you mean to make this `html/template` instead of `text/template`? `html/template` will escape HTML characters, which is likely undesired.

Reply:
Thanks! Nice catch. Fixed.